cypher/cypher-docs/src/docs/dev/ql/schema-index/index.asciidoc

```diff
@@ -34,123 +37,41 @@ Cypher enables the creation of indexes on one or more properties for all nodes t
 * An index that is created on more than one property for any given label is called a _composite index_.
   Differences in the usage patterns between composite and single-property indexes are detailed in the examples below.
 
+[NOTE]
+Use <<query-using, hints>> to force Cypher to use specific indexes.
```

Comment:
Is the word _force_ correct? The default is to add a warning to the results, and only generate an error if the user sets the `cypher.hint_error` to `true`.